cli/config.ts

```diff
@@ -0,0 +1,46 @@
+import { exists, mkdir, readFile, writeFile } from "fs";
+import { homedir } from "os";
+import { join } from "path";
+import { promisify } from "util";
+
+export interface IConfigSettings {
```

Comment:
can you put this in a protobuf definition? or at least put a loud comment on the interface saying that existing members of this interface *must* not be changed or we lose access to those user settings?

Reply:
Added a loud comment. IMO we shouldn't store things as protos unless we need to send them over network? Easy to migrate it if we need to.

Reply:
i don't think i agree with that being the dividing line, I think protos have value in just being a nice clean way to define DTOs, but this case is fine.